jerry-core/debugger/jerry-debugger.h

```diff
@@ -82,12 +107,17 @@ typedef enum
   JERRY_DEBUGGER_EVAL_ERROR_END = 20, /**< last part of eval result when an error is occured */
 
   /* Messages sent by the client to server. */
+    /* The following messages are accepted in both run and breakpoint modes. */
```

Comment:
I didn't like that, but that is an option. If @LaszloLango likes it then I can do it.